cmake/traccc-compiler-options-sycl.cmake

```diff
@@ -7,6 +7,11 @@
 # Include the helper function(s).
 include( traccc-functions )
 
+# Only tweak the flags for the Intel compiler.
+if( NOT "${CMAKE_SYCL_COMPILER_ID}" STREQUAL "IntelLLVM" )
+   return()
+endif()
```

Comment:
Going to leave it up to you to decide, but I would mildly prefer inverting this pattern, e.g. putting the options inside a non-negated `if( "${CMAKE_SYCL_COMPILER_ID}" STREQUAL "IntelLLVM" )` block.

Reply:
My thinking was that I'd change to that once/if we start defining specialized flags for other SYCL compilers as well. Until then I would keep this formalism. :thinking: